server/types/modules/irc-framework.d.ts

```diff
@@ -174,11 +174,18 @@ declare module "irc-framework" {
 
 		changeNick(nick: string): void;
 
-		sendMessage(commandName: string, target: string, message: string): string[];
+		sendMessage(
+			commandName: string,
+			target: string,
+			message: string,
+			tags: Record<string, string> = {}
+		): string[];
 
-		say(target: string, message: string): string[];
+		say(target: string, message: string, tags: Record<string, string> = {}): string[];
 
-		notice(target: string, message: string): string[];
+		notice(target: string, message: string, tags: Record<string, string> = {}): string[];
```

Comment:
overload the functions, tags are optional for all of the above.

Reply:
Done.